modules/Data/Data.yaml

```diff
@@ -91,6 +91,14 @@ enums:
       electrophysiology:
         description: Data generated from an electrophysiology assay.
         meaning: http://purl.obolibrary.org/obo/ERO_0000564
+      Patient Reported Outcome Measures: 
```

Comment:
Hmmmmm, I recommend using the existing `surveyData`. It can be tricky when to know how to break out a concept, but if I were doing reporting this would get merged with `surveyData` anyway.

Reply:
Yeah, that's a fair point!

Reply:
```suggestion
```